src/components/Common/SensitiveInput/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Field } from 'formik';
+import React, { useState } from 'react';
+import EyeOffIcon from '../../../assets/eye-off.svg';
+import EyeIcon from '../../../assets/eye.svg';
```

Comment:
```suggestion
import { EyeIcon, EyeOffIcon } from '@heroicons/react/solid';
```
No need to copy in the SVGs anymore with 8fc71bea08ac5ce3b691ec1de3df1724d91839af 😸 And IMO we should use the solid variant of the icons here, to be consistent with other form buttons (copy and refresh).